CODEOWNERS

```diff
@@ -64,6 +64,10 @@
 
 /Workbooks/LogAnalytics*Workspace/** @microsoft/loganalytics @microsoft/applicationinsights-devtools
 
+/Workbooks/RedisCache/** @microsoft/azure-cache-for-redis @microsoft/applicationinsights-devtools
+/Workbooks/Insights/RedisCache/** @microsoft/azure-cache-for-redis @microsoft/applicationinsights-devtools
+/Workbooks/Resource*Insights/RedisCache/** @microsoft/azure-cache-for-redis @microsoft/applicationinsights-devtools
```

Comment:
These IDs dont match what you have changed below. Expected?

Reply:
Good question, and yes, it's expected.

- The main interesting mismatch is the inclusion of the two paths that don't appear anywhere else in the PR:

  ```codeowners
  /Workbooks/Insights/RedisCache/**
  /Workbooks/Resource*Insights/RedisCache/**
  ```

  These paths are included so the Azure Cache for Redis team can review and merge future changes to the original workbook locations:

  - <https://github.com/microsoft/Application-Insights-Workbooks/blob/master/Workbooks/Insights/RedisCache/Insights.workbook>
  - <https://github.com/microsoft/Application-Insights-Workbooks/blob/master/Workbooks/Resource%20Insights/RedisCache/ResourceInsights.workbook>

  The content in those original workbook locations isn't yet modified at all in this PR, but it will be convenient for the Azure Cache for Redis team to have ownership to adjust those files in the future.

- One other little mismatch are the leading `/` characters that anchor the paths to the repo root in the context of the CODEOWNERS file, but that's also expected.